### PR TITLE
Add ci-tools to manifest

### PR DESCRIPTION
**Pull request by nashif**
There are discussions to add this back to the main zephyr repo with pros and cons. This is one options to make the scripts work for multiple revisions of zephyr and to fix an immediate issue on the 1.14 branch.

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]
+1 for this solution

File: west.yml [hunk 1]
```diff
@@ -56,9 +56,14 @@ manifest:
...
       revision: 45e630d6152824f807d3f919958605c4626cbdff
       path: modules/hal/libmetal
+    - name: mbedtls
+      revision: ca32746072ce3381f1c9ae46ba6cd34c69f8c0ee
+      path: modules/crypto/mbedtls
     - name: mcumgr
       revision: 84934959d2d1722a23b7e7e200191ae4a6f96168
       path: modules/lib/mcumgr
+    - name: net-tools
+      revision: 30b7efa827b04d2e47840716b0372737fe7d6c92
```
Comment (mbolivar) [inline west.yml]: Do you want to stick another commit on top moving net-tools to tools/net-tools?

West won't move the old clone anywhere, though, just FYI, it will just make a new repo.

**Review by mbolivar — APPROVED** [review]

File: west.yml (hunk 1 above)
Comment (nashif) [reply west.yml]: yeah, thought about it, but then wondered what would happen to the old repo, it will be stale, but nothing changes there anyways.
